server/protocols/webadmin/cli/README.md

```diff
@@ -0,0 +1,43 @@
+# Apache James Server/Webadmin command line interface
+
+## Development
+
+'Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface. It also aims at providing a more modern and intuitive interface.
+It is written in Python using Click - a package that supports creating command-line interfaces. Further information about Click can be found [here](https://click.palletsprojects.com).
+
+
+## Run the script
+1. Enter the following command to get Click installed
```

Comment:
this is not enough for an end user / end admin.
We should provide all packaged binaries for CLI (see for example https://stackoverflow.com/questions/5458048/how-to-make-a-python-script-standalone-executable-to-run-without-any-dependency if you really want python), including Windows EXE, deb, rpm, etc. It's a pity to loose the wide availability of Java.

Reply:
I'm really unsatisfied by the use of Python here. In my opinion, there are two decent choices possible:
* a JVM cli (that could use any modern technology like Scala or Kotlin and could be packaged with GraalVM)
* a fat-binary language like Rust (that would not support as many target as a JVM cli but we may live with that limitation)